GraphQL-TSC.md

```diff
@@ -286,6 +286,16 @@ If consensus cannot be reached, a pull request may still be landed after a vote
 
 An exception is made for errata or to update meeting logistics. These may be landed immediately, provided all EasyCLA checks have passed.
 
+### Security policy
+
+The GraphQL TSC is in an elevated position of trust within the GraphQL community. Security concerns that impact repos under the [`graphql` GitHub org](https://github.com/graphql/) (including reference implementations and official tools) may be responsibly disclosed to the TSC via [any current TSC member](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members-1), with the expectation that they will be discussed and triaged by the TSC as a whole.
```

Comment:
@brianwarner should we set up an email alias to have a more clear disclosure guidance?

Reply:
Sure thing. "security@graphql.org"? Maybe going to you and the operations@ alias?

In my experience these tend to attract a lot of spam and emails from bug bounty farms, so it probably makes sense going to someone who can triage before passing it along.

Reply:
This email is up and running if you want to swap this language @benjie

Reply:
Done :+1: